src/plugins/plugin.ts

```diff
@@ -174,6 +168,15 @@ export abstract class Plugin implements ICommandManager {
     return ids
   }
 
+  private async cacheFetchCallback(): Promise<ICommandInfo[]> {
```

Comment:
I would just move this inside of commands as a closure to keep it local to that logic